doc/specifications/drafts/content_edit.md

```diff
@@ -0,0 +1,29 @@
+# Symfony forms content edit
+
+## Synopsis
+repository-forms currently supports creating content without an intermediate draft.
+
+Support must be added for editing content. It implies creating a new version of an existing content,
+rendering a form with the previous version's fields values, and allowing the user to save the draft,
```

Comment:
nitpick: when you create a draft it typically contain the values from previous version initially, so the form won't have to care for this itself, it should just edit the draft all the way until user hits publish.